Gemfile

```diff
@@ -1,5 +1,7 @@
 source 'https://rubygems.org'
 
+gem 'byebug'
+gem 'better_errors'
```

Comment:
can you add `binding_of_caller` as well so we have a console? 

also these gems need to go under the development/test group 

and instead of byebug please add pry-byebug, which is an integration between the two